src/graph.cpp

```diff
@@ -86,7 +86,10 @@ bool Graph::addBidirectionalEdge(const std::string &source, const std::string &d
  * @param source - Id of the source Vertex
  * @param target - Id of the target Vertex
  */
-void Graph::edmondsKarp(const std::string &source, const std::string &target) {
```

Comment:
Keep in mind that an alteration was merged that makes vertex have string ids, your PR shouldn't change that. Replace all ids in your functions with strings